integration/server_test.go

```diff
@@ -1,71 +0,0 @@
-package docker
-
-import "testing"
-
-func TestCreateNumberHostname(t *testing.T) {
-	eng := NewTestEngine(t)
-	defer mkDaemonFromEngine(eng, t).Nuke()
-
-	config, _, _, err := parseRun([]string{"-h", "web.0", unitTestImageID, "echo test"})
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	createTestContainer(eng, config, t)
-}
-
-func TestRunWithTooLowMemoryLimit(t *testing.T) {
```

Comment:
Sorry, it's in the `run` tests now, and there is a PR to move it to API tests...
https://github.com/docker/docker/blob/master/integration-cli/docker_cli_run_test.go#L3500